File: 0168-excel-sheet-column-title/0168-excel-sheet-column-title.java
class Solution {
    public String convertToTitle(int num) {
        StringBuilder str = new StringBuilder();

        while(num > 0){
            num--;
            char ch = (char)('A' + num%26); 

            str.append(ch);
            num /= 26;
        }

        return str.reverse().toString();
    }
}

File: 0168-excel-sheet-column-title/0168-excel-sheet-column-title.cpp
class Solution {
public:
    string convertToTitle(int num) {
        string ans = "";

        while(num > 0){
            num--;

            char ch = (char)('A' + (num%26));
            ans += ch;

            num /= 26;
        }

        reverse(ans.begin(), ans.end());
        return ans;
    }
};